bin/notification.js

```diff
@@ -48,6 +49,28 @@ const queuePopulator = new QueuePopulator(
     zkConfig, kafkaConfig, qpConfig, httpsConfig, mConfig, rConfig, extConfigs);
 
 async.waterfall([
+    done => startProbeServer(qpConfig.probeServer, (err, probeServer) => {
+        if (err) {
+            log.error('error starting probe server', {
+                error: err,
+                method: 'NotificationQueuePopulator::startProbeServer',
+            });
+            done(err);
+            return;
+        }
+        if (probeServer !== undefined) {
+            probeServer.addHandler([DEFAULT_LIVE_ROUTE, DEFAULT_READY_ROUTE],
+                (res, log) => queuePopulator.handleLiveness(res, log)
+            );
+            if (process.env.ENABLE_METRICS_PROBE === 'true') {
+                probeServer.addHandler(
+                    DEFAULT_METRICS_ROUTE,
+                    (res, log) => queuePopulator.handleMetrics(res, log)
+                );
+            }
+        }
+        done();
```

Comment:
To be consistent and return something in both cases (err or not err)
```suggestion
        return done();
```